jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -792,6 +957,14 @@
         )
       (ros::ros-info "[:wait-for-user-input-to-start] received user input: ~a" arm)
       ))
+ ;; defmethod :select-target-bin
+ ;; randomly choose bin for stow task
+  (:select-target-bin
```

Comment:
I think `:select-stow-target-bin` is better name because we can't guess which task `:select-target-bin` is about.